examples/keras_recipes/creating_tfrecords.py

```diff
@@ -0,0 +1,290 @@
+"""
+Title: Creating TFRecords
+Author: [Dimitre Oliveira](https://www.linkedin.com/in/dimitre-oliveira-7a1a0113a/)
+Date created: 2021/02/25
+Last modified: 2021/02/25
+Description: Converting data to TFRecords files.
+"""
+
+"""
+## Introduction
+
+The TFRecord format is a simple format for storing a sequence of binary records,
+converting your data into TFRecords has many advantages:
+
+- **Efficient storage**: the data may end up taking less space and it can also be partitioned into
+multiple files.
+- **Fast I/O**: the data can be read with parallel I/O operations, useful for TPUs or
+multiple hosts.
+- **Self-contained files**: the data can be read from a single source (e.g.
+by contrast, [COCO](https://cocodataset.org/) stores data into two folders,
+"images" and "annotations").
+
+An important use case of TFRecords is training on TPUs: TPUs are fast enough that
+they usually benefit from optimized I/O operations. In addition, TPUs require
+data to be stored remotely (e.g. on Google Cloud Storage), and storing your data as TFRecords
+makes it easier to the load data without batch-downloading it first.
```

Comment:
Suggestion:

- Minor grammatical stuff
- Use `TFRecord` without the "s" (https://www.tensorflow.org/tutorials/load_data/tfrecord?hl=fr-EN)


```suggestion
An important use case of the TFRecord data format  is training on TPUs. First, TPUs are fast
enough to benefit from optimized I/O operations. In addition, TPUs require
data to be stored remotely (e.g. on Google Cloud Storage) and using the TFRecord format
makes it easier to load the data without batch-downloading.
```